oxd-gen-client/src/test/java/io/swagger/client/api/RemoveSiteTest.java

```diff
@@ -0,0 +1,56 @@
+package io.swagger.client.api;
+
+import io.swagger.client.ApiResponse;
+import io.swagger.client.model.RegisterSiteResponseData;
+import io.swagger.client.model.RemoveSiteParams;
+import io.swagger.client.model.UpdateSiteResponse;
+import io.swagger.client.model.UpdateSiteResponseData;
+import org.apache.commons.lang.StringUtils;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import java.util.UUID;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class RemoveSiteTest {
+
+    @Test
+    @Parameters({"opHost", "redirectUrl"})
+    public void testRemoveSite(String opHost, String redirectUrl) throws Exception {
+        final DevelopersApi api = Tester.api();
+        RegisterSiteResponseData response = RegisterSiteTest.registerSite(api, opHost, redirectUrl);
+        String oxdId = response.getOxdId();
+        RemoveSiteParams params = new RemoveSiteParams();
+        params.setOxdId(oxdId);
+        ApiResponse<UpdateSiteResponse> apiResponse =
+                api.removeSiteWithHttpInfo(Tester.getAuthorization(), params);
```

Comment:
Is there any reason to use `api.removeSiteWithHttpInfo()`, if we calls `api.removeSite()` then we don't need to check one line bellow that status code is 200, because swagger client will automatically throw exception if it is not 200, no?

Reply:
You are right. However, if we don't use the method with suffix "WithHttpInfo(...)", then we don't get the status code info from the response unless we parse the exception. As the server might return 404 for resources that are not available(for e.g. non existent oxd id), it might return 5xx for some internal errors, we may not easily put an assert for them unless we've some boilerplate code for parsing the exception trace.  Please let me know about your take on this. Thanks.

Reply:
Here you check 200 status, you don't check it for 404 or 500 or whatever. If you call `api.removeSite()` then swagger will check 200 for you. I mean why do we need this extra code here? Below in `testRemoveSiteWithInvalidOxdId` we definitely need `api.removeSiteWithHttpInfo()` because we will wish to check it for 404 later. But here it looks redundant.